### PR TITLE
include retryable parser which retries with last char removed from command

### PR DESCRIPTION
**Pull request by vadzappa**
since this seems to be an issue in some otherwise valid responses from server (usually invalid login credentials)

amazon aws mails (`imap.mail.eu-west-1.awsapps.com`) have such a response when auth fails, that parser fails to parse it.

example of problematic server response:
```
W1 NO [UNAVAILABLE] Temporary authentication failure. [2023-01-11 10:27:22]
```

where the last part while being timestamp, is wrapped in brackets.
attempt to reparse without last character succeeds though.

pretty hackish way of fix, but

### CODE REVIEW

**Comment by pipedrive-bot-eventsink** [comment]
### Main branch dependencies
[snyk vulnerabilities](https://app.snyk.io/org/tuvi/project/95b15af7-abd1-42c6-b606-06fa48808c35)

File: src/imap.js [hunk 1]
```diff
@@ -47,6 +47,19 @@ const TIMEOUT_SOCKET_LOWER_BOUND = 60000
  */
 const TIMEOUT_SOCKET_MULTIPLIER = 0.1
 
+const retrableParser = (command, opts) => {
```
Comment (mkuuspalu) [inline src/imap.js]: retryableParser* ?

Comment (vadzappa) [reply src/imap.js]: Oh craaap :) true that :)

Comment (vadzappa) [reply src/imap.js]: fixed :)

**Review by mkuuspalu — APPROVED** [review]

**Pull request merged**